docker/context/fish_run_tests.sh

```diff
@@ -3,7 +3,7 @@
 # This script is copied into the root directory of our Docker tests.
 # It is the entry point for running Docker-based tests.
 
-cd ~/fish-build
+cd ~/fish-build || exit 1
```

Comment:
this should probably use `set -e` and `set +e` but I guess this is fine too. It's weird that we `exit` with 0 below